Topic4Services/src/cartServices/ShoppingCartImp.java

```diff
@@ -0,0 +1,57 @@
+package cartServices;
+
+import items.Container;
+import items.Item;
+
+import java.util.List;
+
+import payments.Payment;
+
+public class ShoppingCartImp implements ShoppingCartService {
+
+	private Cart cart;
+	private Container container;
+
+	public ShoppingCartImp(Cart cart, Container container) {

```

Comment:
How are you sending the cart object in the web service from frontend to backend?